<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_server">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>20</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_connect">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>86</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
    <property name="shortcut">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>22</y>
      <width>66</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Server</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_port">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>50</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>53</y>
      <width>66</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_user">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>80</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>83</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_url">
    <property name="geometry">
     <rect>
      <x>86</x>
      <y>120</y>
      <width>191</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>URL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Start at:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>86</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
    <property name="shortcut">
     <string>S</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>161</width>
      <height>57</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Hrs</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Sec</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSpinBox" name="spinBox_hrs"/>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox_mins">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinBox_secs">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>66</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Controls</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_kill">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>86</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Force Kill</string>
    </property>
    <property name="shortcut">
     <string>K</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_n30s">
    <property name="geometry">
     <rect>
      <x>63</x>
      <y>292</y>
      <width>86</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>-30 secs</string>
    </property>
    <property name="shortcut">
     <string>Left</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_p30s">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>292</y>
      <width>86</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>+30 secs</string>
    </property>
    <property name="shortcut">
     <string>Right</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_n10m">
    <property name="geometry">
     <rect>
      <x>63</x>
      <y>326</y>
      <width>86</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>-10 mins</string>
    </property>
    <property name="shortcut">
     <string>Down</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_p10m">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>326</y>
      <width>86</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>+10 mins</string>
    </property>
    <property name="shortcut">
     <string>Up</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pause">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>360</y>
      <width>86</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
    <property name="shortcut">
     <string>Space</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>1000</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
